app/controllers/placements/support/organisations_controller.rb

```diff
@@ -5,39 +5,27 @@ def index
   end
 
   def new
-    organisation
-    organisation_types
+    @organisation_form = OrganisationOnboardingForm.new
   end
 
   def select_type
-    if params_organisation_type == "school"
-      redirect_to new_placements_support_school_path
-    elsif params_organisation_type == "itt_provider"
-      redirect_to new_placements_support_provider_path
+    @organisation_form = OrganisationOnboardingForm.new(
+      organisation_type: organisation_type_param,
+    )
+    if @organisation_form.valid?
+      redirect_to @organisation_form.onboarding_url
     else
-      organisation.errors.add(
-        :organisation_type,
-        t("activerecord.errors.models.organisation.attributes.type.blank"),
-      )
-      organisation_types
       render :new
     end
   end
 
   private
 
-  def organisation_types
-    @organisation_types ||= [
-      OpenStruct.new(id: "itt_provider", name: t("itt_provider")),
-      OpenStruct.new(id: "school", name: t("school")),
-    ]
-  end
-
   def organisation
     @organisation ||= Placements::School.new
   end
```

Comment:
Is this safe to remove now, too?

Reply:
I guess it is :D

Reply:
resolved in [75a5b28](https://github.com/DFE-Digital/itt-mentor-services/pull/107/commits/75a5b28cb435fe5668e151a4db38c61e2cdc4ef5)